webapp/src/app/core/services/block.service.ts

```diff
@@ -43,6 +35,7 @@ export class BlockService {
       case BlockType.Goto: return new GotoBlock(blockType);
       case BlockType.WaitButton: return new WaitButtonBlock(blockType);
       case BlockType.WaitButtonStep: return new WaitButtonStepBlock(blockType);
+      case BlockType.WaitButtonStep: return new WaitButtonStepBlock(blockType);
```

Comment:
Why does this case repeat twice?

Reply:
Oh no, not again. I already removed it. But probably because of reusing code from different branch it came again. I'll remove it.